application/views/main/base.php

```diff
@@ -27,17 +27,49 @@
 			<tr>
 				<td><b>License:</b></td>
 				<td>&nbsp;</td>
-				<td>GNU/GPL</td>
+				<td>MIT</td>
```

Comment:
Kalkun is gpl licensed as per my understanding

http://kalkun.sourceforge.net/about.php

I think somehow licence.txt got added in a bulk pr merge. We should update the license as per the original work

Reply:
It is MIT as per https://github.com/kalkun-sms/Kalkun/blob/devel/license.txt
So what?

Reply:
Maybe it would be better that I let original authors fix all files and clarify the license for everything before continuing to work on that project. Sorry.